#!/usr/bin/env python2.7

PACKAGE='hanse_pingerfollowing'
import roslib; roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()


gen.add("max_amplitude", double_t, 0, "", 4, 0, 1000)
gen.add("base_amplitude", double_t, 0, "", 1, 0, 1000)
gen.add("damping_factor", double_t, 0, "", 1, 0, 1000)
gen.add("damping_exponent", double_t, 0, "", 3, 0, 5)

exit(gen.generate(PACKAGE, "hanse_pingerfollowing", "PingerSimulator"))
